README.md

```diff
@@ -0,0 +1,81 @@
+# Neighbourhood Collaboration Site
+
+We are building a neighbourhood collaboration site and we want to make a system to keep track of people, houses, and addresses of those houses.
+
+## SQL vs NoSQL
+
+Our data will be structured with a predefined schema, therefore we will use an SQL database. SQL databases are relational which will allows us to identify and access data in relation to another piece of data in the database.
+
+## Schema
+
+Table 1 - Owners:
+
+| Owner Name  | Age |
+| ----------- | --- |
+| John Smith  | 45  |
+| Amy Brown   | 36  |
+| Jenny Green | 28  |
+| Tom Hanks   | 54  |
+
+Table 2 - Houses:
+
+| Name        | Street Address      | No. of people in household |
+| ----------- | ------------------- | -------------------------- |
+| John Smith  | 48 Barnaby Street   | 4                          |
+| Amy Brown   | 12 Exeter Road      | 2                          |
+| Jenny Green | 23 Castlehill Way   | 1                          |
+| Tom Hanks   | 8 Coronation Street | 6                          |
+
+Table 3 - Addresses:
+
+| Street Address      | Postcode |
+| ------------------- | -------- |
+| 48 Barnaby Street   | NW1 7PO  |
+| 12 Exeter Road      | NW1 6DT  |
+| 23 Castlehill Way   | NW2 3FT  |
+| 8 Coronation Street | NW1 5PR  |
+
+## API Requests
+
+### Our API should be capable of handling the following requests:
```

Comment:
Nice and comprehensive list of requests. We noticed that you extended them from the original brief which did not include methods of altering or deleting data. Could you please elaborate on your thought process here? Will anyone be able to make those changes? Do you think there might be security issues that could exist here?

Reply:
Thank you for appreciating our list of requests! We had a discussion about update/edit and delete as we thought it would be beneficial for the users to change the database based on the state of the household (for example, having a baby would increase the household members or perhaps if someone moves house, they could delete their entry). We agreed that this could have potential security risks therefore would have to implement some authentication protocol so that users can only edit their own entries.